lib/src/common/model/issues.dart

```diff
@@ -5,7 +5,7 @@ import 'package:json_annotation/json_annotation.dart';
 part 'issues.g.dart';
 
 /// Model class for an issue on the tracker.
-@JsonSerializable()
+@JsonSerializable(fieldRename: FieldRename.snake)
```

Comment:
snake rename is already the default set in build.yaml.
```suggestion
@JsonSerializable()
```

Reply:
I should have an option to warn about redundant settings!